media/static-bundles.json

```diff
@@ -843,6 +850,15 @@
       ],
       "name": "firefox_whatsnew_82"
     },
+    {
+      "files": [
+        "js/firefox/whatsnew/up-to-date.js",
+        "js/libs/spin.min.js",
+        "js/base/send-to-device.js",
```

Comment:
I don't see the send to device widget in the HTML. Is this the code you said you need to remove?

Reply:
I left it as it was in 82 and was the same. Wasn't sure if there was a fallback view or something. 

@craigcook Am I clear to remove this? I can also remove some references in the SCSS for it too. _(Both the 82 JS file and the SCSS file were copied over to 83 as starting points)_

Reply:
82 also doesn't have the widget, so this is redundant in both bundles. I would have copied from 81 which *does* have the widget, and just neglected to remove it (and was also missed in review). We should remove send-to-device.js and spin-min.js from both bundles.

Reply:
Haha. Thank you both. Removing!